keyboards/smolpad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,481 @@
+// Copyright 2025 Wolf Yuan <me at wolf-yuan.dev>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "bitmaps.c"
```

Comment:
As above. And if you choose to keep these in a different file, C source files should be included in compilation via `SRC` in a `rules.mk` file.
```suggestion
```